modules/@apostrophecms/schema/index.js

```diff
@@ -1349,6 +1304,56 @@ module.exports = {
         return schema;
       },
 
+      composeGroups (schema, arrangeFields) {
```

Comment:
Just moved this code to a function because I thought I would need it. I kept it because the initial function is big and complex enough.. Didn't touch any line of the code inside so no risk here.